### PR TITLE
Apply review comments to #110

### PR DESCRIPTION
**Pull request by Swiftb0y**
see commit messages for explanations (feel free to squash into original commit after merging if you prefer that). While trying to apply my review suggestions, I found some of them to not result in improvements, so see those are resolved.

### CODE REVIEW

File: src/pdb/mod.rs [hunk 1]
```diff
@@ -400,11 +394,11 @@ impl BinRead for RowGroup {
         endian: Endian,
         (page_type, page_heap_offset): Self::Args<'_>,
     ) -> BinResult<Self> {
-        let stream_position = reader.stream_position()?;
+        let table_page_end_position = reader.stream_position()?;
```
Comment (Holzhaus) [inline src/pdb/mod.rs:397]: The new name is more descriptive, but unfortunately wrong if there is more than one row group. It's the end position of the current row group.

Maybe 
```suggestion
        let row_group_end_position = reader.stream_position()?;
```

Comment (Swiftb0y) [reply src/pdb/mod.rs]: whoops, yeah you're right. Fixup'd

**Comment by Swiftb0y** [comment]
do you want me to apply the fixup now or whats the next step?

**Comment by Swiftb0y** [comment]
No worries, thank you very much.

**Comment by Swiftb0y** [comment]
Hope you had a nice vacation btw ;)

**Pull request closed without merging**